libp2p/src/builder/phase/quic.rs

```diff
@@ -212,7 +212,7 @@ impl<T: AuthenticatedMultiplexedTransport> SwarmBuilder<super::provider::Tokio,
 
 macro_rules! impl_quic_phase_with_websocket {
     ($providerKebabCase:literal, $providerPascalCase:ty, $websocketStream:ty) => {
-        #[cfg(all(feature = $providerKebabCase, not(target_arch = "wasm32"), feature = "websocket"))]
+        #[cfg(all(feature = $providerKebabCase, not(target_arch = "wasm32"), feature = "websocket", feature = "tcp"))]
```

Comment:
```suggestion
        #[cfg(all(feature = $providerKebabCase, not(target_arch = "wasm32"), feature = "websocket", feature = "tcp", feature = "dns"))]
```

DNS is missing as well, right?

Reply:
That didn't come up as a compile error, I'll look into it.